db/migrate/20150720142931_reregister_archived_whitehall_artefact_routes.rb

```diff
@@ -0,0 +1,14 @@
+class ReregisterArchivedWhitehallArtefactRoutes < Mongoid::Migration
+  def self.up
+    Artefact.where(:state => 'archived', :owning_app => 'whitehall').each do |artefact|
+      # Set state to live (but don't save) so that RoutableArtefact creates the
+      # missing routes for these.
+      artefact.state = 'live'
+      RoutableArtefact.new(artefact).submit(:skip_commit => true)
```

Comment:
I had a similar thought process, but ended with "yeah I should have made them private" :/